crates/kitsune_p2p/dht/src/spacetime/telescoping_times.rs

```diff
@@ -0,0 +1,252 @@
+use super::*;
+
+/// A type which generates a list of exponentially expanding time windows
+/// which fit into a tree structure. See [this document](https://hackmd.io/@hololtd/r1IAIbr5Y)
+/// for the full understanding.
```

Comment:
this hackmd is private to the org - we should move everything in-repo so it can be included in the generated api docs

we can have stub modules to represent this if there isn't a logical item to associate with it

the [doc-comment](https://crates.io/crates/doc-comment) crate can help facilitate separate markdown files being included in doc comments.

Reply:
doc-comment sounds like a solid choice here!